[2025-05-19T20:02:22.743+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-19T20:02:22.802+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_car_3.load_to_greenplum manual__2025-05-19T20:02:10.926163+00:00 [queued]>
[2025-05-19T20:02:22.826+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_car_3.load_to_greenplum manual__2025-05-19T20:02:10.926163+00:00 [queued]>
[2025-05-19T20:02:22.827+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-19T20:02:22.884+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_to_greenplum> on 2025-05-19 20:02:10.926163+00:00
[2025-05-19T20:02:22.909+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=921) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-19T20:02:22.914+0000] {standard_task_runner.py:72} INFO - Started process 930 to run task
[2025-05-19T20:02:22.910+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_car_3', 'load_to_greenplum', 'manual__2025-05-19T20:02:10.926163+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/test_car_3.py', '--cfg-path', '/tmp/tmpsfbih0tm']
[2025-05-19T20:02:22.916+0000] {standard_task_runner.py:105} INFO - Job 23: Subtask load_to_greenplum
[2025-05-19T20:02:23.143+0000] {task_command.py:467} INFO - Running <TaskInstance: test_car_3.load_to_greenplum manual__2025-05-19T20:02:10.926163+00:00 [running]> on host 21ce6642f26a
[2025-05-19T20:02:23.585+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_car_3' AIRFLOW_CTX_TASK_ID='load_to_greenplum' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T20:02:10.926163+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T20:02:10.926163+00:00'
[2025-05-19T20:02:23.594+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-19T20:02:23.668+0000] {test_car_3.py:142} INFO - >>> [START] load_to_greenplum
[2025-05-19T20:02:23.786+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2025-05-19T20:02:23.832+0000] {test_car_3.py:210} INFO - Successfully loaded 14/14 records to GreenPlum
[2025-05-19T20:02:23.834+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-19T20:02:23.866+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-19T20:02:23.868+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_car_3, task_id=load_to_greenplum, run_id=manual__2025-05-19T20:02:10.926163+00:00, execution_date=20250519T200210, start_date=20250519T200222, end_date=20250519T200223
[2025-05-19T20:02:23.977+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-19T20:02:24.036+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-19T20:02:24.044+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
